pkg/types/githubWorkflow.go

```diff
@@ -20,5 +20,6 @@ type push struct {
 type job struct {
 	Permissions map[string]string
 	RunsOn      string `yaml:"runs-on"`
+	Needs		[]string `yaml:"needs,omitempty"`
```

Comment:
weird syntax formatting here with it being spaced out

Reply:
fixed